public/locales/en/translation.json

```diff
@@ -76,5 +76,9 @@
     "canvasActions": "Canvas actions",
     "selectedShapeActions": "Selected shape actions",
     "shapes": "Shapes"
+  },
+  "hints": {
```

Comment:
You're probably out-of-date with master, we just moved the locales from the `public/` folder to the `src/` one

Reply:
I may have updated public first by accident. The locales are already up to date in src. I just removed public locales.